exchanges/bitfinex/bitfinex_wrapper.go

```diff
@@ -199,6 +199,7 @@ func (b *Bitfinex) Setup(exch *config.ExchangeConfig) error {
 		GenerateSubscriptions:            b.GenerateDefaultSubscriptions,
 		Features:                         &b.Features.Supports.WebsocketCapabilities,
 		OrderbookBufferLimit:             exch.WebsocketOrderbookBufferLimit,
+		BufferEnabled:                    exch.WebsocketOrderbookBufferEnabled,
```

Comment:
IDK where to add the comment so i put it here. bitmex if i enable all asset types i get errs where its unable to fetch orderbooks for rest

Reply:
Good pickup: Rest fixed (Asks were desc), as for the index via WebSocket I have updated that to filter it out and not generate a subscription.